README.md

```diff
@@ -27,11 +33,11 @@ Run Instructions for llvm-spirv
 
 llvm-spirv only accepts SPIR 1.2/2.0 or LLVM bitcode following a [SPIR-V friendly format](https://github.com/KhronosGroup/SPIRV-LLVM/blob/khronos/spirv-3.6.1/docs/SPIRVRepresentationInLLVM.rst).
 
-The [Open Source SPIR producer](https://github.com/KhronosGroup/SPIR) can be used to compile OpenCL 1.2/2.0 C source code to SPIR 1.2/2.0.
+The [Khronos OpenCL C compiler](https://github.com/KhronosGroup/SPIR) can be used to compile OpenCL 1.2/2.0 C source code to SPIR 1.2/2.0 or SPIR-V. It has three branches: spir_12, spir_20_provisional, and spirv-1.0, corresponding to SPIR 1.2, SPIR 2.0, and SPIR-V, respectively. The spirv-1.0 branch emits SPIR-V directly.
```

Comment:
I think we should promote using spirv-1.0 branch or mention that spir_12/spir_20_provisional compiler doesn't work for all  the cases (e.g. sampler not always correctly translated to SPIR-V as well as access qualifiers for images and pipes). Do you agree?